espnet/nets/pytorch_backend/transformer/attention.py

```diff
@@ -5,7 +5,7 @@
 from torch import nn
 
 
-MIN_VALUE = float(numpy.finfo(numpy.float32).min)
+MIN_VALUE = float(numpy.finfo(numpy.float16).min)
```

Comment:
I changed this line because the float16 tensor's `torch.masked_fill` does not accept the `float32.min` value because it is out-of numerical range. For strong compatibility, I will create a new function like
```python
def mask_by_min(x, mask):
    min_value = float(numpy.finfo(x.dtype).min)
    return x.maked_fill(min_value)
```